test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -84,12 +63,27 @@ def test_duplicate_scheduler_name(self):
         try:
             self.server.manager(MGR_CMD_CREATE,
                                 SCHED,
+                                {'sched_port': '15052'},
                                 id="TestCommonSched")
         except PbsManagerError as e:
             if self.server.get_op_mode() == PTL_CLI:
                 self.assertTrue(
                     'qmgr: Error (15211) returned from server' in e.msg[1])
 
+    def test_invalid_sched_port(self):
+        """
+        Test setting invalid port.
+        """
+        try:
+            self.server.manager(MGR_CMD_SET, SCHED,
+                                {'sched_port': 'asdf'},
+                                id="TestCommonSched",
+                                runas=ROOT_USER)
+        except PbsManagerError as e:
+            err_msg = "Illegal attribute or resource value"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
```

Comment:
Either use an else on the try/except or an expect() checking op=UNSET after the block.  If the sched_port was successfully changed, this test will still pass.